collections/_pages/active_members.html

```diff
@@ -0,0 +1,12 @@
+---
+layout: page
+#title:  "Members | Layer5"
```

Comment:
I think you have commented out the title by mistake over here, and it should be `Active Members | Layer5`

Reply:
I think the title is not necessary here since I hardcode it in the active_member.html file.